modular_skyrat/master_files/code/modules/events/spacevine.dm

```diff
@@ -27,49 +27,53 @@
 		new /mob/living/simple_animal/hostile/venus_human_trap(spawning_turf)
 
 /datum/spacevine_mutation
+	/// The name of the mutation, it will be displayed in the examine of any vine objects.
 	var/name = ""
+	/// How severe the plant it, think of it as the usefullness. Used in probability calculations.
 	var/severity = 1
-	var/hue
+	/// The color of the plant
+	var/plant_color
+	/// What kind of mutation is it, good, bad, really bad?
```

Comment:
Some sexy documentation. Although, for `plant_color`, wouldn't it be better to have a default color, just in case?

Reply:
No color just means the vines don't get an override